inc/em_base.h

```diff
@@ -24,6 +24,8 @@
 #include <uuid/uuid.h>
 #include "ec_base.h"
 
+#include <vector>
```

Comment:
em_base.h is used in OneWifi which uses gcc to compile. vector cannot be used

Reply:
Hi @amarnathhullur yeap, I am looking for another solution for this segfault (please take notice that it is just draft for cli checking)